zubhub_frontend/zubhub/src/views/activity_details/MakeActivity.js

```diff
@@ -0,0 +1,126 @@
+  const classes = makeStyles(activityDefailsStyles)();
+  const commonClasses = makeStyles(styles)();
+  return (
+    <>
+      <Container style={{ backgroundColor: 'white', border: 'solid 1.5rem white'}} className={classes.pdfStyle}>
+        <Container className={classes.pdfMainLogo}>
+          <Avatar className={classes.pdfLogoStyle} src={images.logo} alt={'Faridah_ux'} />
+        </Container>
+        <Box className={classes.verticalSpace}></Box>
+        <Container className={classes.justifyPdf}>
+          <Typography className={classes.pdfTitle}>{activity?.title}</Typography>
+          <Container className={`${classes.creatorProfilePdfStyle}`}>
+            <Avatar className={classes.creatorAvatarStyle} src={creator?.avatar} alt={'Faridah_ux'} />
+            <Typography>{activity?.creators[0].username}</Typography>
+          </Container>
+        </Container>
+        <Box className={classes.verticalSpace}></Box>
+        <Typography className={`${classes.pdfSubtitle}`}>Introduction</Typography>
+        <Typography className={classes.textStyle}>
+          {activity.introduction ? <div dangerouslySetInnerHTML={{ __html: activity.introduction }} /> : ''}
+        </Typography>
+        {activity.images?.length > 0 && (
+          <div className={classes.imageStyle}>
+            {activity.images.map((img, index) => (
+              <img
+                key={index}
+                src={img.image?.file_url}
+                alt={`Image ${index}`}
+                className={classes.imageIndividualStyle}
+              />
+            ))}
+          </div>
+        )}
+        <Box className={classes.verticalSpace}></Box>
+
+        <Typography className={`${classes.pdfSubtitle}`}>Category</Typography>
+        <div className={clsx(commonClasses.displayFlex, commonClasses.gap, commonClasses.flexWrap)}>
+          {activity.category?.map((cat, index) => (
+            <Pill key={index} text={cat} />
+          ))}
+        </div>
+        <Box className={classes.verticalSpace}></Box>
+
+        <Typography className={`${classes.pdfSubtitle}`}>Class Grade</Typography>
+        <div className={clsx(commonClasses.displayFlex, commonClasses.gap, commonClasses.flexWrap)}>
+          <Pill text={activity.class_grade} />
+        </div>
+        <Box className={classes.verticalSpace}></Box>
+
+        <Typography className={`${classes.pdfSubtitle}`}>Materials Used</Typography>
+        <Typography
+          className={`${classes.textStyle} ${classes.customTypographyStyle}`}
+          theme={'bubble'}
+          readOnly={true}
+          value={activity.materials_used || ''}
+        >
+          {activity.materials_used ? <div dangerouslySetInnerHTML={{ __html: activity.materials_used }} /> : ''}
+        </Typography>
+        {console.log(activity.materials_used_image)}
+        <div className={classes.imageStyle}>
+          <img
+            src={activity.materials_used_image?.file_url}
+            // alt={`Image ${index}`}
```

Comment:
if we don't need the alt here then we should remove it instead of commenting it out